terraform/bootstrap/remote-state/production/main.tf

```diff
@@ -0,0 +1,8 @@
+module "production_tf_state" {
+    source = "../../../modules/remote-state"
+    environment = "production"
+}
+
+variable "environment" {
+    description = "The environment (development/staging/production)"
```

Comment:
```suggestion
    description = "The environment (development/staging/production)"
    default = "production"
```

Reply:
i'm going to remove `development` from the text as well 😅 , since if you can reach those terraform file means youre never `env=developement`